app/models/container_group.rb

```diff
@@ -37,6 +38,17 @@ def ready_condition_status
     ready_condition.try(:status) || 'None'
   end
 
+  def container_states_summary
+    summary = Hash.new(0)
+    containers.each { |container| summary[container.state.to_sym] += 1 }
+    summary
+  end
+
+  def running_containers_summary
+    running = containers.where(:state => 'running').count
+    "#{running}/#{containers.count}"
```

Comment:
This seems to do two queries. You could build this on top of `container_states_summary`.